[2024-08-12T08:55:24.008+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-12T08:55:24.051+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_2024-08-12.decrypt_the_data scheduled__2024-05-07T00:00:00+00:00 [queued]>
[2024-08-12T08:55:24.057+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_2024-08-12.decrypt_the_data scheduled__2024-05-07T00:00:00+00:00 [queued]>
[2024-08-12T08:55:24.057+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-12T08:55:24.084+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): decrypt_the_data> on 2024-05-07 00:00:00+00:00
[2024-08-12T08:55:24.087+0000] {standard_task_runner.py:63} INFO - Started process 4036 to run task
[2024-08-12T08:55:24.090+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_2024-08-12', 'decrypt_the_data', 'scheduled__2024-05-07T00:00:00+00:00', '--job-id', '2619', '--raw', '--subdir', 'DAGS_FOLDER/final_etl.py', '--cfg-path', '/tmp/tmpkc40h1qd']
[2024-08-12T08:55:24.093+0000] {standard_task_runner.py:91} INFO - Job 2619: Subtask decrypt_the_data
[2024-08-12T08:55:24.145+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_2024-08-12.decrypt_the_data scheduled__2024-05-07T00:00:00+00:00 [running]> on host 88a939613e48
[2024-08-12T08:55:24.253+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_2024-08-12' AIRFLOW_CTX_TASK_ID='decrypt_the_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-07T00:00:00+00:00'
[2024-08-12T08:55:24.253+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-12T08:55:24.261+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-12T08:55:24.262+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/data/decryption_utils.py']
[2024-08-12T08:55:24.269+0000] {subprocess.py:86} INFO - Output:
[2024-08-12T08:55:24.362+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-12T08:55:24.363+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-12T08:55:24.466+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_2024-08-12, task_id=decrypt_the_data, run_id=scheduled__2024-05-07T00:00:00+00:00, execution_date=20240507T000000, start_date=20240812T085524, end_date=20240812T085524
[2024-08-12T08:55:24.541+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-12T08:55:24.566+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-12T08:55:24.593+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
